src/richie/apps/courses/templatetags/extra_tags.py

```diff
@@ -229,12 +234,29 @@ def course_enrollment_widget_props(context):
     """
     course_run = context["run"]
 
+    profile_urls = json.loads(
+        context.get("AUTHENTICATION", {}).get("profile_urls", "{}")
+    )
+    dashboard_link = profile_urls.get("dashboard", {}).get("action")
+
+    starts_in_message = None
+    if course_run.start > timezone.now():
+        course_start = arrow.get(course_run.start)
+        humanized_course_start = course_start.humanize(
+            arrow.now(), locale=to_locale(get_language())
+        )
+        starts_in_message = _("The course will start {:s}").format(
+            humanized_course_start
+        )
```

Comment:
I think you should name the variable and add a comment for translators explaining what it looks like because it is not obvious:
```suggestion
        # Translators: delay indicates when the course will start as a duration. In english the string will be "The course will start in 3 days"
        starts_in_message = _("The course will start {delay:s}").format(
            delay=humanized_course_start
        )
```

See https://docs.djangoproject.com/fr/2.2/topics/i18n/translation/#comments-for-translators